build/python/backend/Dockerfile

```diff
@@ -7,8 +7,13 @@ ARG YARA_PYTHON_VERSION=4.1.3
 ARG CAPA_VERSION=3.0.3
 ARG EXIFTOOL_VERSION=12.38
 
-# expat: CVE-2022-23852 & CVE-2022-23990; mount & util-linux: CVE-2021-3996 & CVE-2021-3995
-RUN apt-get update -y && apt-get install -y expat=2.4.1-2ubuntu0.3 mount=2.36.1-8ubuntu2.2 util-linux=2.36.1-8ubuntu2.2
+RUN apt-get update -y && apt-get install -y \
+    # expat: CVE-2022-23852 & CVE-2022-23990;
```

Comment:
Might be a good idea to create a dependency file that lists out all of these, and then install from there; I have a feeling this list will grow in the future!

Reply:
👍 I'll look into doing that for the next round!